### PR TITLE
New version: Microsoft.PowerBI version 2.118.828.0

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110551&drop=dogfoodAlpha

### CODE REVIEW

**Comment by Trenly** [comment]
Usually takes some time for PowerBI CDNs to update

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335215 2023-06-23 20:17:26.878 [CLI ] Package hash verification failed. SHA256 in manifest [8a9f1b4549c36d7882f2e9ef1c91788bc4c58c24fda9d8f320809d22f505f236] does not match download [14272ea7454690d77845562091408302835da50fe6d5af8d32ccb77c4a0d9ee7]

File: manifests/m/Microsoft/PowerBI/2.118.828.0/Microsoft.PowerBI.installer.yaml [hunk 1]
```diff
@@ -22,6 +22,6 @@ UpgradeBehavior: install
 Installers:
 - Architecture: x64
   InstallerUrl: https://download.microsoft.com/download/8/8/0/880BCA75-79DD-466A-927D-1ABF1F5454B0/PBIDesktopSetup_x64.exe
-  InstallerSha256: 14272EA7454690D77845562091408302835DA50FE6D5AF8D32CCB77C4A0D9EE7
+  InstallerSha256: 8A9F1B4549C36D7882F2E9EF1C91788BC4C58C24FDA9D8F320809D22F505F236
```
Comment (stephengillie) [inline manifests/m/Microsoft/PowerBI/2.118.828.0/Microsoft.PowerBI.installer.yaml:25]:
```suggestion
  InstallerSha256: 14272EA7454690D77845562091408302835DA50FE6D5AF8D32CCB77C4A0D9EE7
```

Comment (stephengillie) [reply manifests/m/Microsoft/PowerBI/2.118.828.0/Microsoft.PowerBI.installer.yaml:25]: Was it reverted?

Comment (Trenly) [reply manifests/m/Microsoft/PowerBI/2.118.828.0/Microsoft.PowerBI.installer.yaml:25]: @stephengillie - That is the hash of the old version. The issue here is with the PowerBI CDN, and is known. It takes many hours for all of their CDN endpoints to receive the new version, so I'd anticipate that the hash issue will resolve itself over the weekend

**Review by ItzLevvie — APPROVED** [review]

**Comment by elreymon** [comment]
Error is present already

2023-07-06 15:49:58.353 [CLI ] Package hash verification failed. **SHA256 in manifest** [8a9f1b4549c36d7882f2e9ef1c91788bc4c58c24fda9d8f320809d22f505f236] **does not match download** [511e86ab13e204a00b73301e70deb40e323f6c809277b8200df7160ea2a1bc4b]
2023-07-06 15:49:58.353 [CLI ] Terminating context: 0x8a150011 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DownloadFlow.cpp:15d

Please review. Thanks in advance.

**Pull request merged**